src/random_test.cpp

```diff
@@ -1709,7 +1709,9 @@ void load_default_data(Table *table, Thd1 *thd) {
   thd->ddl_query = false;
   static int initial_records =
       options->at(Option::INITIAL_RECORDS_IN_TABLE)->getInt();
-  int rec = rand_int(initial_records);
+  int rec = options->at(Option::EXACT_INITIAL_RECORDS)->getBool()
```

Comment:
What happens if a user passes --jldll. The load_default_data() will not be called and no records would be inserted in tables

Reply:
jlddl is to created ddl and exit.

Reply:
Ok. You can ignore this comment.